_layouts/midnight-train.html

```diff
@@ -441,11 +446,15 @@
 			font-family: "heading";
 			text-transform: uppercase;
 		}
-		 h3 {
+
+		h3
+		{
 			width: 100%;
 			max-width: 50rem;
-			margin: 0.5rem auto 0.5rem;
-			font-family: "heading";
```

Comment:
Why did you remove this?

Reply:
I found the heading font wasn't good looking at such small sizes, so I swapped it out for the body font bolded and capatilized.